dll/win32/shell32/wine/control.c

```diff
@@ -865,6 +865,105 @@ static	void	Control_DoLaunch(CPanel* panel, HWND hWnd, LPCWSTR wszCmd)
     *   "a path\foo.cpl"
     */
 {
+#ifdef __REACTOS__
+    signed sp = -1;
+    CPlApplet *applet;
+    LPCWSTR wszFirstCommaPosition = NULL, wszSecondCommaPosition = NULL;
+    LPCWSTR wszLastUnquotedSpacePosition = NULL;
+    LPWSTR buffer, buffer2;
```

Comment:
`buffer` is the name from the original code (which didn't use very descriptive names in my opinion), I've named the other buffer `buffer2`, because I didn't know how to name it, and decided a less descriptive, but correct name is better than an incorrect one (see one of my other responses).